typescript/packages/plugins/spl-token/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@goat-sdk/plugin-spl-token",
```

Comment:
should we namespace with solana somewhere

Reply:
its kinda weird cuz spl is solana program library... u think @goat-sdk/solana-spl-token? or @goat-sdk/solana-token

Reply:
actually just leave as is, we'll just need to figrue out namespacing for v1 later